st2api/st2api/controllers/resource.py

```diff
@@ -50,7 +50,9 @@ class ResourceController(rest.RestController):
     from_model_kwargs = {}
 
     # Maximum value of limit which can be specified by user
-    max_limit = cfg.CONF.api.max_page_size
+    @property
+    def maxlimit(self):
```

Comment:
I've fixed property name, but using `git blame` in this case (and really, most cases) is a far superior way to figure out why some or other line was needed.